cssom-view-1/Overview.bs

```diff
@@ -1929,12 +1938,14 @@ Whenever a <a>visual viewport</a> gets scrolled (whether in response to user int
 
 When asked to <dfn export for=Document>run the scroll steps</dfn> for a {{Document}} <var>doc</var>, run these steps:
 
+1. Run the steps to <a spec="css-scroll-snap-2">dispatch pending snapchanging events</a> for <var>doc</var>.
 1. For each item <var>target</var> in <var>doc</var>'s <a>pending scroll event targets</a>,
     in the order they were added to the list, run these substeps:
 
     1. If <var>target</var> is a {{Document}}, <a>fire an event</a> named <a event>scroll</a> that bubbles at <var>target</var>.
     1. Otherwise, <a>fire an event</a> named <a event>scroll</a> at <var>target</var>.
 1. Empty <var>doc</var>'s <a>pending scroll event targets</a>.
+1. Run the steps to <a spec="css-scroll-snap-2">dispatch pending snapchanged events</a> for <var>doc</var>.
```

Comment:
I wonder if there can be ordering issues if we always dispatch snapchanged after snapchanging. I'm thinking of cases where you may have a snapchanged queued for a finished scroll or layout re-snap, followed by subsequent snapchanging for a newly started scroll in which case dispatching the snapchanged afterwards implies that the snapchanging operation is complete. Thoughts?

Reply:
To avoid lumping too much complexity into this one PR, perhaps this is worth a followup issue / PR (and maybe a test case)?

Reply:
Sounds good, I'd prefer that too.